src/Core/AdminConsole/Services/Implementations/PolicyService.cs

```diff
@@ -348,11 +351,35 @@ await _mailService.SendOrganizationUserRemovedForPolicySingleOrgEmailAsync(
                         }
                     }
                     break;
+                case PolicyType.FreeFamiliesSponsorshipPolicy:
```

Comment:
:warning: We're moving to a new paradigm for adding policies.  It's currently in an in-between state. 

The new implementation creates an implementation of [IPolicyValidator](https://github.com/bitwarden/server/blob/main/src/Core/AdminConsole/OrganizationFeatures/Policies/IPolicyValidator.cs) when there are validation rules or side-effects from updating a policy.

Since this new type has the side-effect of sending an email, you would need to create a Validator for it.  [TwoFactorAuthenticationPolicyValidator](https://github.com/bitwarden/server/blob/main/src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyValidators/TwoFactorAuthenticationPolicyValidator.cs) is a good example to match your policy. It has the side-effect of when being enabled, it revokes all non-compliant users.

If you have any additional questions, feel free to reach out and thanks!